pokedex-with-bit/app.py

```diff
@@ -29,30 +35,91 @@ def index():
 
 
 # for search
-def search():
+def query():
     f = Flow.load_config('flows/query.yml')
 
     with f:
         f.block()
 
 
-if __name__ == '__main__':
-    if len(sys.argv) < 2:
-        print('choose between "index" and "search" mode')
-        exit(1)
-    if sys.argv[1] == 'index':
-        config()
-        workspace = os.environ['JINA_WORKSPACE']
+def index_restful(num_docs):
+    config()
+    f = Flow().load_config('flows/index.yml')
+
+    with f:
+        data_path = os.path.join(os.path.dirname(__file__), os.environ.get('JINA_DATA_FILE', None))
+        url = f'http://0.0.0.0:{f.port_expose}/index'
+
+        input_docs = _input_lines(
+            filepath=data_path,
+            size=num_docs,
+            read_mode='r',
+        )
+        data_json = {'data': [Document(text=text).dict() for text in input_docs]}
+        r = requests.post(url, json=data_json)
+        if r.status_code != 200:
+            raise Exception(f'api request failed, url: {url}, status: {r.status_code}, content: {r.content}')
+
+
+def query_restful():
+    config()
+    f = Flow().load_config("flows/query.yml")
+    f.use_rest_gateway()
+    with f:
+        f.block()
+
+# for test before put into docker
+def dryrun():
+    f = Flow().load_config("flows/index.yml")
+    with f:
+        pass
+
+
+@click.command()
+@click.option('--task', '-t',
+              type=click.Choice(['index', 'query', 'index_restful', 'query_restful', 'dryrun'], case_sensitive=False))
+@click.option('--num_docs_query', '-n', default=100)
+@click.option('--num_docs_index', '-n', default=600)
+def main(task, num_docs_query, num_docs_index):
+    config(task)
```

Comment:
This leads to the error
```suggestion
    config(task)
```

Reply:
Removed code